services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/claimprovider/GA4GHPassportClaimProvider.java

```diff
@@ -112,7 +136,22 @@ GA4GHVisaPayload getVisaForAccessRequirement(String arId, String subject, String
 		} else {
 			throw new IllegalArgumentException("Unexpected AccessRequirement type: "+concreteType);
 		}
-		return result;
+		return visa;
+	}
+	
+	String visaAsJWS(GA4GHVisa visa, String subject) {
```

Comment:
Can we factor out the signature with this type of key (oidcSignaturePrivateKey)?